### DISCUSSION TITLE
Is it possible to train 2 different people?

### DISCUSSION THREAD

**Discussion by Jpkovas** [d8ahazard/sd_dreambooth_extension#396] [Q&A]
I’ve been testing how to train 2 different people in a single model, but can’t get this to work

**Comment by illumnat** [comment]
I haven't tried it with this extension, but I did train it on 2 separate people with TheLastBen's colab: [https://github.com/TheLastBen/fast-stable-diffusion](url). I would think it would be similar here.

I trained on photos of two friends of mine who are theater professors/actors so I had plenty of photos to pull from... about 24 of each. The  photos of her were named "dntbkrcpl (1).png, dntbkrcpl (2).png" etc. Similarly, the photos of him were named "edbkrcpl.png (1)" and so forth. I put all of the photos for training in a single folder.

Dreambooth recognizes the differing filename as a different subject.

Then, when I wanted to create images of them, I could use "dntbkrcpl" and "edbkrcpl" in my prompts singly or together. For example, here's a prompt that had both of them together and the resulting image. (I added some film grain in Photoshop with the Nik Collection plugins but otherwise unedited.)

𝘢 𝘣𝘭𝘢𝘤𝘬 𝘢𝘯𝘥 𝘸𝘩𝘪𝘵𝘦 𝘱𝘩𝘰𝘵𝘰 𝘰𝘧 (𝘦𝘥𝘣𝘬𝘳𝘤𝘱𝘭 𝘸𝘦𝘢𝘳𝘪𝘯𝘨 𝘢 𝘭𝘢𝘣 𝘤𝘰𝘢𝘵) 𝘢𝘯𝘥 (𝘥𝘯𝘵𝘣𝘬𝘳𝘤𝘱𝘭 𝘸𝘦𝘢𝘳𝘪𝘯𝘨 𝘢 𝘭𝘢𝘣 𝘤𝘰𝘢𝘵) 𝘪𝘯 𝘢 𝘮𝘢𝘥 𝘴𝘤𝘪𝘦𝘯𝘵𝘪𝘴𝘵 𝘭𝘢𝘣, 𝘑𝘢𝘮𝘦𝘴 𝘞𝘩𝘢𝘭𝘦, (𝘭𝘰𝘸 𝘢𝘯𝘨𝘭𝘦), 𝘍𝘳𝘢𝘯𝘬𝘦𝘯𝘴𝘵𝘦𝘪𝘯, 1930𝘴, 𝘥𝘳𝘢𝘮𝘢𝘵𝘪𝘤 𝘭𝘪𝘨𝘩𝘵𝘪𝘯𝘨, 𝘷𝘰𝘭𝘶𝘮𝘦𝘵𝘳𝘪𝘤 𝘭𝘪𝘨𝘩𝘵𝘪𝘯𝘨, 40𝘮𝘮 𝘧/2.8, 𝘨𝘳𝘢𝘪𝘯𝘺, 𝘧𝘪𝘭𝘮 𝘥𝘢𝘮𝘢𝘨𝘦, 𝘰𝘭𝘥 𝘮𝘰𝘷𝘪𝘦, 𝘤𝘪𝘯𝘦𝘮𝘢𝘵𝘪𝘤, 𝘯𝘰𝘪𝘳, 𝘥𝘢𝘳𝘬, 𝘴𝘱𝘰𝘰𝘬𝘺

𝘕𝘦𝘨𝘢𝘵𝘪𝘷𝘦 𝘱𝘳𝘰𝘮𝘱𝘵: 𝘧𝘢𝘬𝘦, 𝘤𝘢𝘳𝘵𝘰𝘰𝘯, 𝘱𝘢𝘪𝘯𝘵𝘪𝘯𝘨

𝘚𝘵𝘦𝘱𝘴: 100, 𝘚𝘢𝘮𝘱𝘭𝘦𝘳: 𝘋𝘗𝘔++ 2𝘔 𝘒𝘢𝘳𝘳𝘢𝘴, 𝘊𝘍𝘎 𝘴𝘤𝘢𝘭𝘦: 9, 𝘚𝘦𝘦𝘥: 1428485975, 𝘚𝘪𝘻𝘦: 768𝘹512, 𝘔𝘰𝘥𝘦𝘭 𝘩𝘢𝘴𝘩: 3675𝘢327, 𝘈𝘦𝘴𝘵𝘩𝘦𝘵𝘪𝘤 𝘓𝘙: 0.0001, 𝘈𝘦𝘴𝘵𝘩𝘦𝘵𝘪𝘤 𝘸𝘦𝘪𝘨𝘩𝘵: 0.1, 𝘈𝘦𝘴𝘵𝘩𝘦𝘵𝘪𝘤 𝘴𝘵𝘦𝘱𝘴: 5, 𝘈𝘦𝘴𝘵𝘩𝘦𝘵𝘪𝘤 𝘦𝘮𝘣𝘦𝘥𝘥𝘪𝘯𝘨: 𝘧𝘳𝘬𝘯𝘴𝘵𝘦𝘪𝘯, 𝘈𝘦𝘴𝘵𝘩𝘦𝘵𝘪𝘤 𝘴𝘭𝘦𝘳𝘱: 𝘍𝘢𝘭𝘴𝘦, 𝘈𝘦𝘴𝘵𝘩𝘦𝘵𝘪𝘤 𝘵𝘦𝘹𝘵: , 𝘈𝘦𝘴𝘵𝘩𝘦𝘵𝘪𝘤 𝘵𝘦𝘹𝘵 𝘯𝘦𝘨𝘢𝘵𝘪𝘷𝘦: 𝘍𝘢𝘭𝘴𝘦, 𝘈𝘦𝘴𝘵𝘩𝘦𝘵𝘪𝘤 𝘴𝘭𝘦𝘳𝘱 𝘢𝘯𝘨𝘭𝘦: 0.1

![image](https://user-images.githubusercontent.com/10290174/205465849-d481b180-a8a7-475c-b16a-c98d57900ec2.png)

Here's one of just him using "edbkrcpl" by itself in a prompt. (unfortunately, I don't seem to have saved the prompt on this one.) Again, this I added film grain in this one but otherwise unedited.

![image](https://user-images.githubusercontent.com/10290174/205465956-bfa45977-cc32-4439-9920-e0dd262214ba.png)

Good luck!!
